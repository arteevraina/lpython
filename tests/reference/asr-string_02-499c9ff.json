{
    "basename": "asr-string_02-499c9ff",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/string_02.py",
    "infile_hash": "ed6511565e893791a4bd8ea0b4750817bab13cd6dc0731332127bf58",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-string_02-499c9ff.stderr",
    "stderr_hash": "85593e60baa6c6f930b7486a92e41fac9943108966abdacb7acb39ef",
    "returncode": 2
}