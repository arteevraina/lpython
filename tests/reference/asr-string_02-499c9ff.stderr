warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/errors/string_02.py:6:15
  |
6 |     res:str = x.join(p)
  |               ^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
